### DISCUSSION TITLE
zh/docs/permission/adapter

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2158] [Docs comments]
# zh/docs/permission/adapter

Config adapter and basic CRUD to policy

https://casdoor.org/zh/docs/permission/adapter

**Comment by Frederic-Zhou** [comment]
What do v0~v5 represent?

**Reply by hsluoyz** [reply]
See: https://casbin.org/docs/adapters#how-to-write-an-adapter

**Reply by Frederic-Zhou** [reply]
Thx！！！
